test/service.ts

```diff
@@ -1315,9 +1379,10 @@ describe('GrpcService', () => {
 
       beforeEach(() => {
         delete grpcService.grpcCredentials;
-        grpcService.getGrpcCredentials_ = callback => {
+        grpcService.getGrpcCredentials_ = (callback: Function) => {
           callback(null, authClient);
         };
+        // tslint:disable-next-line no-any
```

Comment:
Is anything on the next line evaluating to `any`? 👀

Reply:
oops! thanks